src/tests/class_tests/openms/data/newMergerTest_out.idXML

```diff
@@ -11,19 +11,15 @@
 		<ProteinIdentification score_type="" higher_score_better="false" significance_threshold="0" >
 			<ProteinHit id="PH_0" accession="A" score="0.258519381284714" sequence="" >
 				<UserParam type="string" name="target_decoy" value="target"/>
-				<UserParam type="float" name="XTandem_score" value="0"/>
```

Comment:
I would leave this in, since unfortunately this is how it would come out of our pipelines.

Reply:
It comes from score switching and new score/fdr calculations. We always store the old score, even if it is meaningless.

Reply:
But yeah maybe you can leave it out. It is just a corner case to test maybe.